api/v1/verticadb_types.go

```diff
@@ -117,6 +117,11 @@ type VerticaDBSpec struct {
 	// options are to create a new database or revive an existing one.
 	InitPolicy CommunalInitPolicy `json:"initPolicy"`
 
+	// +kubebuilder:validation:Optional
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors="urn:alm:descriptor:com.tectonic.ui:advanced"
+	// The restore point policy specifies which restore point among the archive to restore from
```

Comment:
In future we may use part of the RestorePoint struct to know what archive to create a save point in. It won't necessarily only be used for bootstraping a database with a restore. Can we change the language slightly?
```suggestion
	// Specifies the restore point details to use with this instance of the VerticaDB.
```